bin/oref0-setup.sh

```diff
@@ -113,6 +122,17 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
         echo -n TTY $ttyport
     fi
     echo " it is."
+	# 
+	if [[ -n "$ttyport" ]]; then
+		read -p "Are you using a USA (US) or world wide (WW) Medtronic pump? If US, press enter, otherwise enter WW. " -r
+		radio_locale=$REPLY
+		if [[ $radio_locale =~ ^[Ww][Ww]$ ]]; then
```

Comment:
You should use the `${radio_locale,,} =~ "ww"` syntax here to do a case insensitive comparison.